ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/BasicAction.java

```diff
@@ -3386,6 +3386,19 @@ private final boolean checkChildren (boolean isCommit)
                     tsLogger.i18NLogger.warn_coordinator_BasicAction_57(get_uid());
                 } else {
                     tsLogger.i18NLogger.warn_coordinator_BasicAction_58(get_uid());
+
```

Comment:
Actually if you step through to the actual log call it does

    if (isEnabled(level))

so I withdraw my comment.